OpenRA.Mods.Cnc/Traits/SupportPowers/AttackOrderPower.cs

```diff
@@ -47,11 +47,15 @@ public override void Activate(Actor self, Order order, SupportPowerManager manag
 			attack.AttackTarget(Target.FromCell(self.World, order.TargetLocation), false, false, true);
 		}
 
-		void INotifyCreated.Created(Actor self)
+		protected override void Created(Actor self)
 		{
 			attack = self.Trait<AttackBase>();
+
+			base.Created(self);
 		}
 
+		void INotifyCreated.Created(Actor self) { Created(self); }
```

Comment:
The `Created` overrides will be called by the parent class, so these explicit implementations (and the interface on these subclasses) are no longer needed and can be removed.